test/fixtures/index.js

```diff
@@ -32,6 +32,7 @@ describe('custom resolver', () => {
           ],
           pattern: this.pattern,
           customResolver: this.customResolver,
+          nameFormatter: (path) => basename(path).replace(/\.[^/.]+$/, ''), // Remove file extension
```

Comment:
Please use [`path.parse().name`](https://nodejs.org/api/path.html#path_path_parse_path) instead :)

Reply:
Ah, thanks for the tip!